data/hasRoadsWithNoSpeedLimit.yml

```diff
@@ -0,0 +1,9 @@
+# Whether there exist any roads in the country on which there is no speed limit
+# that is, roads where vehicles may go as fast as they like
+# or, where maxspeed=none is a valid value

```

Comment:
Though in such cases it can be still shown and input can be handled by notes.

So maybe just change "Whether there exist any roads" to "Whether there exist any regular/public/?? roads"

Reply:
(The fact that Nurburgring is in Germany is not really relevant, I guess)

Makes sense, I've added the word "public" to clarify that.